front/cypress/e2e/form_builder/create_and_fill_survey.cy.ts

```diff
@@ -485,4 +485,77 @@ describe('Survey builder', () => {
     cy.get('.e2e-modal-close-button').click();
     cy.get('#e2e-modal-container').should('have.length', 0);
   });
+
+  it('shows validation errors when previous pages are referenced', () => {
```

Comment:
@EdwinKato One case is not covered by this test: `shows validation error when the same page is referenced'`